capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -82,12 +78,29 @@ public double getEventLongitude() {
   public List<Review> getReviews() {
     return reviews;
   }
-  public void setDatastoreID(Long datastoreID) {
-    this.datastoreID = datastoreID;
+
+  public String getOrganizationName() {
+    return organizationName;
+  }
+
+  public Long getOrganizationId() {
+    return organizationId;
   }
 
-  public void setOrganization(Organization organization) {
-    this.organization = organization;
+  public String getEventDescription() {
+    return eventDescription;
+  }
+
+  public Boolean getFoodAvaliable() {
+    return foodAvaliable;
+  }
+
+  public Boolean getRequiredFee() {
+    return requiredFee;
+  }
+
+  public void setDatastoreID(Long datastoreID) {
```

Comment:
I'd suggest agreeing on a naming convention as a team about whether it's "Id" or "ID". Pure camel-case is more common in Java ("Id"), but it's more important to be consistent within a project.